### PR TITLE
Add JSON linter to Travis

### LINKED ISSUE #39: Add JSON linter to Travis
To test for well-formedness of data in pull requests like #38 automatically.

### PR DESCRIPTION
**Pull request by abitrolly**
Fixes #39

### CODE REVIEW

**Comment by staaas** [comment]
can we use [demjson](http://deron.meranda.us/python/demjson/) instead of jsonlint?

i'd like to be able to check everything locally via `python manage.py test` which calls `tox` under the hood. using python tools can simplify the setup in this case.

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,13 @@ addons:
...
-install: "pip install tox"
+install:
+  - pip install tox
+  - npm install jsonlint -g
 before_script:
   - psql -c 'create database filmfest;' -U postgres
-script: tox -e "py27-{sqlite,pg}"
+script:
+  - # --- Run JSON linter ---
+  - find . -name "*.json" -type f -exec jsonlint -q "{}" \; -exec /bin/echo "{}" \;
```
Comment (staaas) [inline .travis.yml]: in an ideal world i'd like each json file to become a pytest test.

https://github.com/tholo/pytest-flake8/blob/master/pytest_flake8.py works this way.

**Comment by abitrolly** [comment]
I don't mind. It requires writing custom script for `tox`, because its command-line `jsonlint` executable doesn't work on Windows.

File: .travis.yml (hunk 1 above)
Comment (abitrolly) [reply .travis.yml]: That's a long way to go. =)

**Comment by abitrolly** [comment]
@nott should be fine now.

**Comment by davojta** [comment]
LGTM

**Comment by abitrolly** [comment]
@davojta, you can merge it sooner, so go ahead if you have the time.

**Pull request merged**